### ISSUE TITLE
Unused import in 0.16.1 when building the tests

### ISSUE THREAD

**Issue by johanneskastl** [veeso/termscp#312]
Hi,

building termscp for openSUSE spits out these warnings when building the tests:

```
[  552s] warning: unused import: `pretty_assertions::assert_eq`
[  552s]   --> src/system/keys/keyringstorage.rs:79:9
[  552s]    |
[  552s] 79 |     use pretty_assertions::assert_eq;
[  552s]    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[  552s]    |
[  552s]    = note: `#[warn(unused_imports)]` on by default
[  552s] 
[  552s] warning: unused import: `whoami::username`
[  552s]   --> src/system/keys/keyringstorage.rs:80:9
[  552s]    |
[  552s] 80 |     use whoami::username;
[  552s]    |         ^^^^^^^^^^^^^^^^
[  552s] 
[  552s] warning: unused import: `super::*`
[  552s]   --> src/system/keys/keyringstorage.rs:82:9
[  552s]    |
[  552s] 82 |     use super::*;
[  552s]    |         ^^^^^^^^
[  552s] 
[  568s] warning: `termscp` (bin "termscp" test) generated 3 warnings (run `cargo fix --bin "termscp" --tests` to apply 3 suggestions)
[  568s]     Finished `test` profile [unoptimized + debuginfo] target(s) in 3m 09s
[  568s]      Running unittests src/main.rs (target/debug/deps/termscp-efa4c291569df56b)
[  568s] 
[  568s] running 214 tests
```

**Comment by veeso** [comment]
Hi,

fixed 099e2154 nothing concerning since are imports in test units

**Comment by johanneskastl** [comment]
Wow, that was quick. Thanks!
